src/defs_version.h

```diff
@@ -53,6 +53,7 @@
     // In that case, this value is used
 
     // READ ABOVE CAREFULLY BEFORE CHANGING!
-    // USE 3 DIGITS VERSION ONLY /!\ _
-    #define MIXXX_VERSION "2.3.0-alpha-pre"
+    // USE 3 DIGITS VERSION ONLY !
```

Comment:
This line belongs to where the version is actually defined, i.e. version.h.

Reply:
The reason i moved define out of the file in the first place is because it's very hard to do text processing in the nix configuration language. I tried different approaches for 2 days, trying to extract version from git, parsing the current file. This was the only solution that was fast and somehow elegant. Therefor, no comments in the version.h file.